datascience/species-prediction/README.adoc

```diff
@@ -21,7 +20,7 @@ In order to get started, make sure that you have the following components:
 +
 ```
 $ cd <PATH/TO/SPRING-CLOUD-DATAFLOW>
-$ java -jar spring-cloud-dataflow-server-local/target/spring-cloud-dataflow-server-local-1.0.0.BUILD-SNAPSHOT.jar
+$ java -jar spring-cloud-dataflow-server-local/target/spring-cloud-dataflow-server-local-[VERSION].BUILD-SNAPSHOT.jar
```

Comment:
There is some inconsistency IMHO, where we use `<>` on the `cd` line, and then `[]` to indicate versions. We should choose just one (angle brackets, I believe)